### PR TITLE
CB-21553 Implement CM policy configuration to be Fips compliant

### PR DESCRIPTION
**Pull request by doktoric**
See detailed description in the commit message.

### CODE REVIEW

**Review by bergerdenes — APPROVED** [review]

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java [hunk 1]
```diff
@@ -95,6 +95,72 @@ public class ClouderaManagerSetupService implements ClusterSetupService {
 
     private static final String SDX_ENTERPRISE_DATALAKE_TEXT = "Enterprise SDX";
 
+    private static final String OPENSSL_INTERMEDIATE2018 =
```
Comment (biharitomi) [inline cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java:98]: Can we move this constants to a separate constant class as it is almost 100 lines of Fedramp/Gov related configs?

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java [hunk 2]
```diff
@@ -323,10 +389,35 @@ public void publishPolicy(String template, boolean govCloud) {
                 apiConfigPolicy.setName("FISMA Policy For Login Banner");
 
                 List<ApiConfigEnforcement> apiConfigEnforcements = new ArrayList<>();
-                ApiConfigEnforcement apiConfigEnforcement = new ApiConfigEnforcement();
-                apiConfigEnforcement.setLabel("LOGIN_BANNER");
-                apiConfigEnforcement.setDefaultValue(banner);
-                apiConfigEnforcements.add(apiConfigEnforcement);
+
+                ApiConfigEnforcement loginBannerEnforcement = new ApiConfigEnforcement();
```
Comment (biharitomi) [inline cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java:393]: Can we have a separate policy factory or policy configurer service for this Gov specific logic with UT?

**Review by biharitomi — CHANGES_REQUESTED** [review]

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java (hunk 2 above)
Comment (doktoric) [reply cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java:393]: will do this. thanks Tomi

**Review by biharitomi — APPROVED** [review]
LGTM, thanks

**Pull request merged**